src/main/java/org/opentripplanner/gtfs/mapping/TransitModeMapper.java

```diff
@@ -51,33 +51,24 @@ public static TransitMode mapMode(int routeType) {
       return null;
     } else if (routeType >= 1600 && routeType < 1700) { //Self drive
       return TransitMode.BUS;
-    } else if (routeType >= 1700 && routeType < 1800) { //Miscellaneous Service
+      // 1700 is mapped to CARPOOL further down
+    } else if (routeType >= 1701 && routeType < 1800) { //Miscellaneous Service
```

Comment:
I'm against using 1700, as other feeds might use it for miscellaneous and would not want it associated to something specific like car pooling.

Reply:
What should 1700 be mapped to?

Reply:
I spoke to @hbruch who produces these GTFS feeds and he also suggested the 15xx range, so therefor it's three against one :).

I suggest the range 1550-1560 for carpool in order to leave "room" for other taxi modes.